docs/rules/no-extraneous-dependencies.md

```diff
@@ -31,7 +37,13 @@ If you are using JavaScript configuration (e.g., `.eslintrc.js`), then you can u
 "import/no-extraneous-dependencies": ["error", {"devDependencies": /test|spec/}]
 ```
 
-When using a regular expression, the setting will be activated if the name of the file being linted matches the given regular expression. For example, the above configurations will allow the import of `devDependencies` in files whose names include `test` or `spec`.
+When using a glob or regular expression, the setting will be activated if the name of the file being linted matches the given glob or regular expression. In addition, you can use an array to specify multiple globs or regular expressions:
+
+```js
+"import/no-extraneous-dependencies": ["error", {"devDependencies": [/test/, '/spec/', '*.test.js', '*.spec.js']}]
+```
+
+When using an array of globs or regular expressions, the setting will be activated if the name of the file being linted matches a single glob or regular expression in the array.
```

Comment:
Dang it. Back to the drawing board.

Maybe we should only support arrays for globs? But then what is excluded? Just things that contain an exclamation mark? In the case of `['*.test.js', '*.spec.js']`, I'd want that to be additive. In the case of `['*.test.js', '!**/fixtures/**']`, I'd want that to be subtractive.

This is approaching a point of complexity that may not be best for this plugin to handle. In your multiple test and fixture directory example we're already approaching a status of, "really, now?" At least IMHO.

A thought occurs. Maybe allow a function for configuration? Pass the name to the function and expect a boolean back. This plugin could support the most basic use cases and then people who need more complex functionality can implement it themselves.

Re: your closing question... Is what a glob? The string containing `<root>`? Not sure.